bpf/lib/clustermesh.h

```diff
@@ -4,6 +4,12 @@
 
 #include "lib/utils.h"
 
+#ifndef __CLUSTERMESH_IDENTITY__
+#define __CLUSTERMESH_IDENTITY__
+DECLARE_CONFIG(__u32, cluster_id_max, "Max number of clusters that can be connected in clustermesh")
```

Comment:
I think we may have our first exception for declaring a NODE_CONFIG in a feature header. Given the nature of the feature, there's no way to decouple identity lookup from clustermesh or avoid including the header in any program that touches a packet, since identity lookup will always be necessary. Feel free to convert this, that will take away a lot of the boilerplate, and we don't want to forget to set this anywhere.

Nit: our documentation and marketing always mentions as `Cluster Mesh` or `Clustermesh`, so I'd like this to be consistent in docstrings.
```suggestion
DECLARE_CONFIG(__u32, cluster_id_max, "Max number of clusters that can be connected in Clustermesh")
```

Reply:
Super, done!